frontend/package.json

```diff
@@ -10,8 +10,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@apollo/client": "^3.8.1",
```

Comment:
are we using graphql?

Reply:
I use @apollo/client for globalState on the front end but not as a DB.